### DISCUSSION TITLE
How do I disable SuiteAdvanced?

### DISCUSSION THREAD

**Discussion by fiedag** [burkybang/SuiteAdvanced-Feedback#146] [Q&A] [answered]
I am constantly taking screenshots of my NetSuite forms, saved searches, reports etc. and sending them to clients. who are NetSuite key users or end users.  

With SuiteAdvanced enabled, these screenshots now have a cosmetically different look. Minor but significant to users. It is going to be a distraction to them, I just know it will. They will puzzle over whether we are even in the same system and whether this is possibly a factor in the problem they are having.

Would it be easy enough to add an on/off toggle?  Right now it seems I would have to "remove from Chrome".

**Comment by burkybang** [comment, marked as answer]
There are multiple ways to disable SuiteAdvanced. In the SuiteAdvanced menu at the top left of NetSuite, navigate to SuiteAdvanced -> Disable.

**Reply by burkybang** [reply]
![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/4654263f-a553-48ce-8a24-ad7439e15642)
